.setup/pip/system_requirements.txt

```diff
@@ -11,8 +11,7 @@
 # For QR bulk upload
 opencv-python-headless==4.10.0.84
 
-# jsonschema & jsonref & pytz & tzlocal
-pytz==2023.3.post1 # Submitty-util specific.
+# jsonschema & jsonref & tzlocal
```

Comment:
Is this line needed now?

Reply:
To be honest I do not know why it is even there. We can probably just remove it, but I didn't want to mess with it too much.